client/src/components/FeedbackObject/index.js

```diff
@@ -7,159 +7,148 @@ import Button from "../Button/Button";
-									name={property}
-									value={currentFeedback[property]}
-									onChange={handleEdit}
-									disabled={!isEditing}
-								/>
-							</div>
-						);
-					}
-					if (property === "text") {
-						return (
-							<div className="previous-feedback-text" key={index}>
-								<p className="feedback-input-head">Feedback:</p>
-								<textarea
-									className="previous-feedback-text-input"
-									name={property}
-									value={currentFeedback[property]}
-									onChange={handleEdit}
-									disabled={!isEditing}
-								/>
-							</div>
-						);
-					}
-					if (property === "mentor") {
-						return (
-							<div className="previous-feedback-mentor" key={index}>
-								<p className="feedback-input-head">Given by:</p>
-								<input
-									className="previous-feedback-mentor-input"
-									name={property}
-									value={currentFeedback[property]}
-									onChange={handleEdit}
-									disabled
-								/>
-							</div>
-						);
-					}
-					if (property === "time") {
-						return (
-							<div className="previous-feedback-time" key={index}>
-								<input
-									className="previous-feedback-time-input"
-									name={property}
-									value={timeDifference(Date.now(), currentFeedback[property])}
-									onChange={handleEdit}
-									disabled
-								/>
-							</div>
-						);
-					}
-				})}
-			<div className="edit-delete-buttons">
-				<Button
-					content={isEditing ? "Save" : "Edit"}
-					handleClick={() => {
-						setIsEditing(!isEditing);
-						isEditing && saveFeedback(currentFeedback.id);
-					}}
-				/>
-				<div>
-					<Button
-						content="Delete"
-						handleClick={() => {
-							alert("DELETE");
-							handleDelete(currentFeedback.id);
-							updateFeedback();
-						}}
-					/>
-				</div>
-			</div>
-		</div>
-	);
+  return (
+    <div className="prev-feedback-list">
+      {Object.keys(currentFeedback).length &&
+        Object.keys(currentFeedback).map((property, index) => {
+          if (property === "module") {
+            return (
+              <div className="feedback-title-and-module">
+                <div className="feedback-title" key={index}>
+                    <textarea
+                      className="feedback-title-textarea"
+                      name="title"
+                      value={currentFeedback.title}
+                      onChange={handleEdit}
+                      disabled={!isEditing}
+                    />
+                </div>
+                <div className="feedback-module" key={index}>
+                    <h6 className="title-module">Module:</h6>
+                    {!isEditing ? (
+                      <p className="modules-dropdown">
+                        {currentFeedback.module}
+                      </p>
+                    ) : (
+                      <select
+                        name={property}
+                        value={currentFeedback[property]}
```

Comment:
You might want to change the "property" in value and name to be consistent with the rest of elements?

Reply:
Done